src/main/java/frc/lib/util/COTSFalconSwerveConstants.java

```diff
@@ -0,0 +1,115 @@
+package frc.lib.util;
+
+import edu.wpi.first.math.util.Units;
+
+/* Contains values and required settings for common COTS swerve modules. */
+public class COTSFalconSwerveConstants {
+    public final double wheelDiameter;
+    public final double wheelCircumference;
+    public final double angleGearRatio;
+    public final double driveGearRatio;
+    public final double angleKP;
+    public final double angleKI;
+    public final double angleKD;
+    public final double angleKF;
+    public final boolean driveMotorInvert;
+    public final boolean angleMotorInvert;
+    public final boolean canCoderInvert;
+
+    public COTSFalconSwerveConstants(double wheelDiameter, double angleGearRatio, double driveGearRatio, double angleKP, double angleKI, double angleKD, double angleKF, boolean driveMotorInvert, boolean angleMotorInvert, boolean canCoderInvert){
```

Comment:
This ought to be private honestly. It's pretty clear that users of this want to call one of the statics below rather than come along and say they want a totally custom set of constants.

Reply:
Thank you! I agree, fixed.